src/hooks/useCioAutocomplete.ts

```diff
@@ -117,6 +117,24 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
       const { index, sectionId } = getItemPosition({ item, items });
       const sectionItemTestId = `cio-item-${sectionId?.replace(' ', '')}`;
 
+      // Products always have links, Search Suggestions with getSearchResultsUrl have links
+      const hasLink =
+        item.data?.url || (item.section === 'Search Suggestions' && getSearchResultsUrl);
```

Comment:
Not required, but better it's explicitly false
```suggestion
      const hasLink =
        item.data?.url || (item.section === 'Search Suggestions' && getSearchResultsUrl) || false;
```